apps/web/components/sections/project/media-player.tsx

```
@@ -0,0 +1,84 @@
+								alt={item.alt || 'Thumbnail'}
+								className="w-full h-full object-cover"
+							/>
+						) : (
+							<div className="text-gray-500 flex items-center justify-center w-full h-full">
+								<svg
+									xmlns="http://www.w3.org/2000/svg"
+									className="h-6 w-6"
+									fill="none"
+									viewBox="0 0 24 24"
+									stroke="currentColor"
+								>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)